azext_iot/custom.py

```diff
@@ -800,3 +815,279 @@ def iot_device_upload_file(client, device_id, hub_name, file_path, content_type,
         custom_sdk.post_file_notification(device_id, container['correlationId'])
     except errors.ErrorDetailsException as e:
         raise CLIError(e)
+
+# DPS Enrollments
+
+API_VERSION = '2017-11-15'
```

Comment:
Consider storing API_VERSION in the SDK Client scope, so you don't have to provide it for every function. The other SDK's will do this.

Look at the custom SDK for a short example.

https://github.com/Azure/azure-iot-cli-extension/blob/master/azext_iot/custom_sdk/custom_api.py#L47